Cargo.toml

```diff
@@ -31,3 +31,8 @@ nalgebra = { version = "0.32", default-features = false, features = ["std"] }
 
 [package.metadata.docs.rs]
 all-features = true
+
+[features]
+default = ["numpy-1", "numpy-2"]
```

Comment:
What is the cost to enabling a "support level" if it is never used? Since we have to do the check in any case, do the extra features really carry their weight?